odd_collector/adapters/airbyte/mappers/connections.py

```diff
@@ -0,0 +1,132 @@
+import logging
+from odd_models.models import DataEntity, DataEntityType, DataTransformer
+from oddrn_generator import AirbyteGenerator
+from .oddrn import generate_connection_oddrn, generate_dataset_oddrn
+from ..api import AirbyteApi, OddPlatformApi
+
+
+async def map_connection(
+    connection: dict,
+    oddrn_gen: AirbyteGenerator,
+    airbyte_api: AirbyteApi,
```

Comment:
Mapper should be a pure function.  Move side effects as calling API's to adapter file.